package.json

```diff
@@ -4,7 +4,7 @@
     "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
     "test-watch": "meteor test --driver-package practicalmeteor:mocha",
     "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha",
-    "lint": "eslint .",
+    "lint": "meteor node node_modules/eslint/bin/eslint.js .",
```

Comment:
👍
On Wed, 11 May 2016 at 10:07 PM, Archimedes Trajano <
notifications@github.com> wrote:

> In package.json
> <https://github.com/meteor/todos/pull/132#discussion_r62965651>:
>
> > @@ -4,7 +4,7 @@
> >      "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
> >      "test-watch": "meteor test --driver-package practicalmeteor:mocha",
> >      "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha",
> > -    "lint": "eslint .",
> > +    "lint": "meteor node node_modules/eslint/bin/eslint.js .",
>
> Ah I see what you mean I just tried it on boilerplate
>
> "gulp": "gulp",
> "lint": "gulp lint",
> "tidy": "gulp tidy"
>
> Seems to work well even after npm uninstall -g gulp (I verified that gulp
> command itself will fail`)
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/meteor/todos/pull/132/files/f226294f4dd14e74b87b82512dc300576c2c1ce9#r62965651>
>